integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -754,6 +841,15 @@ fn test_resharding_v3_base(params: TestReshardingParameters) {
             }
             trie_sanity_check.assert_state_sanity(&clients, expected_num_shards);
             latest_block_height.set(tip.height);
+            let shard_layout =
+                client.epoch_manager.get_epoch_config(&tip.epoch_id).unwrap().shard_layout;
+            println!(
```

Comment:
nit: `tracing::info` and use the structured logging by setting fields rather than putting them in string
```
tracing::info!(
  last_block_hash=?tip.last_block_hash,
  ...
  "block"
);
```